### PR TITLE
Include the encryption example in the distribution package

### PR DESCRIPTION
**Pull request by emanuelez**
Also adds it for monkey tests

### CODE REVIEW

**Comment by emanuelez** [comment]
@kneth @bmunkholm @cmelchior

File: tools/monkey-examples.sh [hunk 1]
```diff
@@ -20,11 +20,6 @@ for example in examples/*/ ; do
     applicationId=$(grep applicationId "${example}build.gradle" | cut -d \" -f 2 | cut -d \' -f 2)
     project=$(basename ${example})
 
-    if [ ${project} == "encryptionExample" ] ; then
```
Comment (kneth) [inline tools/monkey-examples.sh]: Nice job

File: distribution/RealmEncryptionExample/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile 'io.realm:realm-android:0.77.0'
```
Comment (cmelchior) [inline distribution/RealmEncryptionExample/app/build.gradle]: This is different than the other examples where we use: 

```
tasks.preBuild {
    dependsOn ":realm:androidJar"
}

dependencies {
    compile files("../../realm/build/libs/realm-${version}.jar")
}
```

**Comment by cmelchior** [comment]
Do we have any speed measurements between a encrypted and non-encrypted Realm?

File: distribution/RealmEncryptionExample/app/build.gradle (hunk 2 above)
Comment (emanuelez) [inline distribution/RealmEncryptionExample/app/build.gradle]: This is the distribution example so it uses the jcenter version

Comment (cmelchior) [inline distribution/RealmEncryptionExample/app/build.gradle]: Doh :+1:

**Comment by bmunkholm** [comment]
We have for cocoa. If I remember correctly it's around < 20% slower. That should be roughly the same for Java. But worth measuring since the binding overhead may be different.

**Comment by bmunkholm** [comment]
:+1:

**Pull request merged**